### ISSUE TITLE
`Octo pr list`: Not possible to filter by branch or author in PR picker

### ISSUE THREAD

**Issue by alexkalderimis** [pwntester/octo.nvim#551]
## Issue Description

Type: *feature request*

### Describe what happened (or what feature you want)

In the PR picker, there is no way to filter the PRs by author, either when typing or as arguments, and the only way to filter by branch name is as arguments.

This is more important for larger projects where there may be many PRs, and being able to use the generic sorter match against author and branch names would help significantly for projects that follow the convention of putting issue identifiers in the branch name.

### Describe what you expected to happen

When I use `Octo pr list`, and then type my user name, it should show only PRs I created.

**Comment by pwntester** [comment]
asdfasdfasdfasdf
asdfasdf
asdfasdfj

**Comment by natdm** [comment]
Agreed here. Adding the author to the line item in the fzf/telescope list would solve this for some teams.

**Comment by wilkinson4** [comment]
I've been using this command
`:Octo search author:USERNAME is:pr is:open`
